src/lib/utils/index.ts

```diff
@@ -13,6 +16,26 @@ export function isSameAddress(address1: string, address2: string): boolean {
   return address1.toLowerCase() === address2.toLowerCase()
 }
 
+// TODO: Settle on one token typing approach?
```

Comment:
Personally, I think it's ok that the app has its own token type - as we're adding a lot of extra properties that wouldn't be useful in tokenlists. I think it's common to have a kind of adapter pattern for these things. But if annoying we could look into maybe handling the extra properties in different ways. I think lots of them are kind of obsolete now.